src/store/info/types.js

```diff
@@ -9,7 +9,7 @@ export const GET_MEMBER_SHARE = 'GET_MEMBER_SHARE'
 export const GET_POOL_SHARE = 'GET_POOL_SHARE'
 export const GET_SHARE_OF_BASE_AMAOUNT = 'GET_SHARE_OF_BASE_AMAOUNT'
 export const GET_SHARE_OF_TOKEN_AMAOUNT = 'GET_SHARE_OF_TOKEN_AMAOUNT'
-export const GET_POOL_SHARE_ASSYM = 'GET_POOLSHARE_ASSYM'
+export const GET_POOL_SHARE_ASSYM = 'GET_POOL_SHARE_ASSYM'
```

Comment:
did you change also in the reducer of the info? I cannot find that change

Reply:
I changed it to match the reducer 👍 
Just a little one I think we both missed